WordPress/gutenberg
# List View: Allow the component to show a custom "more" menu.

## What?
This allows the ListView to accept a custom "more" menu component. Closes https://github.com/WordPress/gutenberg/issues/46991

## Why?
When building the Navigation List View, we discovered a need for different "more" menus in different contexts. This will allow us to reuse the list view component in the Navigation block rather than having a copy of the ListView.

## How?
Adds a new prop to ListView which specifies the custom "more" menu.

## Testing Instructions
1. Open the site editor
2. Open the list view
3. Select the "more" menu next an item in the list view
4. Check that the options are the same as on trunk
5. Open the post editor
6. Open the list view
7. Select the "more" menu next an item in the list view
8. Check that there is no "more" menu.

### Testing Instructions for Keyboard
As above.

### Note
We will need to add the "more" menu back to the post editor before we merge this.
